README.md

```diff
@@ -92,34 +140,32 @@ If you wish to create a layout with [two thirds, one third styling](https://desi
 as follows:
 
 ```html
-@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.{HmrcStandardHeader, HmrcStandardFooter, HmrcScripts, HmrcHead, HmrcLanguageSelectHelper}
 @import views.html.helper.CSPNonce
 
 @this(
-  govukLayout: GovukLayout,
-  hmrcStandardHeader: HmrcStandardHeader,
-  hmrcStandardFooter: HmrcStandardFooter,
-  head: HmrcHead,
-  hmrcLanguageSelectHelper: HmrcLanguageSelectHelper,
-  scripts: HmrcScripts,
-  twoThirdsOneThirdMainContent: TwoThirdsOneThirdMainContent
+        govukLayout: GovukLayout,
+        head: Head,
+        scripts: Scripts,
+        twoThirdsOneThirdMainContent: TwoThirdsOneThirdMainContent
 )
 
-@(pageTitle: String, beforeContent: Option[Html] = None, isWelshTranslationAvailable: Boolean = true)(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
+@(pageTitle: Option[String] = None,
+        beforeContentBlock: Option[Html] = None
+)(contentBlock: Html)(implicit request: Request[_], messages: Messages)
 
 @sidebar = {
-  <h2 class="govuk-heading-xl">This is my sidebar</h2>
-  <p class="govuk-body">There is my sidebar content</p>
+    <h2 class="govuk-heading-xl">This is my sidebar</h2>
+    <p class="govuk-body">There is my sidebar content</p>
 }
 
 @govukLayout(
-  pageTitle = Some(pageTitle),
-  headBlock = Some(head(nonce = CSPNonce.get)),
-  headerBlock = Some(hmrcStandardHeader(displayHmrcBanner = true)),
-  scriptsBlock = Some(scripts(nonce = CSPNonce.get)),
-  beforeContentBlock = if(isWelshTranslationAvailable) Some(hmrcLanguageSelectHelper()) else None,
-  footerBlock = Some(hmrcStandardFooter()),
-  mainContentLayout = Some(twoThirdsOneThirdMainContent(sidebar))
+    pageTitle = pageTitle,
+    headBlock = Some(head()),
+    beforeContentBlock = beforeContentBlock,
```

Comment:
I changed this to be consistent with the advice on the standalone instructions page, given the hmrc helpers are not available in play-frontend-govuk.